File: server/token_server/java/src/test/java/com/netease/im/rtctoken/RestApiAuthTest.java
package com.netease.im.rtctoken;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class RestApiAuthTest {
    @Test
    public void testChecksum() throws Exception {
       assertEquals("192bdbdad337836e6213aec1d93186aae9771c39", RestApiAuth.getChecksum("c00000000000", "1234567890", 1697168455));
    }
}

File: server/token_server/nodejs/test/RestAPIAuthTest.js
const test = require('node:test');
const assert = require('node:assert');


const { GetChecksum } = require('../src/RestApiAuth.js');

test('GetChecksum', (t) => {
    var appSecret = "c00000000000";
    var nonce = "1234567890";
    var curTime = 1697168455;
    assert.equal("192bdbdad337836e6213aec1d93186aae9771c39", GetChecksum(appSecret, nonce, curTime));
});